### PR TITLE
nice handle for Voron 2.4

### PR DESCRIPTION
**Pull request by 1-0-R**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
A nice handle for Voron 2.4 (2020 extrusion with 6mm panel)

#### How was it tested? 
I tested it myself.

### CODE REVIEW

**Comment by MapleLeafMakers** [comment]
CAD Files should be provided in .STEP or .SCAD format.   You can include the fcstd *in addition* to one of those formats.

File: printer_mods/1-0-R/handle/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+![insert heat set inserts](images/inner_part_with_heat_set_inserts.jpg)
+
+2. put T-nuts in aluminium extrusion (2 T-nuts in top slot, 3 T-nuts in side slot, coarsely adjust hole distance)
+
+![T-nuts in extrusion](images/extrusion_with_tnuts.jpg)
+
+3. attach inner handle part to top side of the extrusion
+![attach inner handle](images/extrusion_with_inner_part_mounted.jpg)
+
+4. attach middle and outer handle part to the side of the extrusion; do not tighten M5 scews jet
```
[the remark this answers is not in the document]
Comment (1-0-R) [reply printer_mods/1-0-R/handle/README.md]: fixed typos

**Comment by 1-0-R** [comment]
added .STEP model

**Comment by MapleLeafMakers** [comment]
This looks good, though I would recommend renaming `handle_middle_part(A).stl` to `[a]_handle_middle_part.stl` to match the Voron naming conventions for accent colors.

**Comment by 1-0-R** [comment]
renamed STL file to match naming standard

**Comment by MapleLeafMakers** [comment]
LGTM

**Review by cbc02009 — APPROVED** [review]

**Comment by cbc02009** [comment]
@1-0-R looks like some the of the STLs need to be repaired before this can be merged. check the CI output for which files specifically.

**Comment by ccatlett1984** [comment]
Would it be possible to get a variant of this for 3mm panels?

**Comment by 1-0-R** [comment]
> Would it be possible to get a variant of this for 3mm panels?

The handle is designed for 3mm panels ( 3mm panel + 3mm foam tape).

**Comment by ccatlett1984** [comment]
Thanks for the reply. I was looking to use this on my Trident, which specs 1mm foam tape on the top panel. I attempted to edit the step file, Fusion is not happy with the geometry. I guess I can just switch to the thicker foam tape. Thanks again for the reply.

**Pull request merged**